test/unit/lifecycle/properties.js

```diff
@@ -29,35 +29,35 @@ describe('lifecycle/property', function () {
   it('should return an function', function () {
     expect(propertiesInit()).to.be.a('function');
   });
-  
+
   it('should not leak options to other definitions', function () {
     const elem = element().skate({
       properties: {
-        test1: { 
-          attribute: true, 
-          default: 'test1', 
+        test1: {
+          attribute: true,
+          default: 'test1',
```

Comment:
> Ok, I think I understand. So if default and initial were both specified, the return value for initial would be used?

That's correct.

> Only thing would be documentation around the differences between default, initial and created, as I imagine this would be initially confusing.

Totes. I'll leave this in PR until we can document it then. It'll finally get me to document those options :)